src/applications/mhv/secure-messaging/components/MessageThread/MessageThreadBody.jsx

```diff
@@ -7,7 +7,7 @@ const MessageThreadBody = props => {
 
   const componentDecorator = (href, linkText) => (
     <a href={href} target="_blank" rel="noreferrer">
-      {linkText}
+      {`${linkText} (opens a new tab)`}

```

Comment:
please confirm with UCD, I saw a not somewhere by a11y member where they mentioned that article `a` is not needed here in this expression

Reply:
reached out to UCD and the final decision is to say "(opens in new tab)".  I'll update it.